### PR TITLE
Remove all 2dii refs

### LINKED ISSUE #295: Remove all references to 2DII
- [ ] DESCRIPTION file
- [ ] GH actions
- [ ] (Maybe) License file(s)
- [ ] Function docs
- [ ] README
- [ ] pkgdown links
- [ ] vignettes
- [ ] update NEWS.md (if relevant)

### PR DESCRIPTION
**Pull request by jdhoffa**
Remove all 2dii references from: 
* DESCRIPTION
* README
* GH actions
* Function links and deprecation links
* pkgdown
* vignettes

Some links were left that correspond to, for example, blog posts that never migrated to RMI. 
(e.g. https://2degreesinvesting.github.io/posts/2022-03-02-ald-becomes-abcd/ )

Closes #295

### CODE REVIEW

**Comment by jdhoffa** [comment]
Failing check seems to be unrelated to this PR, see #297

File: README.md [hunk 1]
```diff
@@ -1,52 +1,51 @@
...
-```r
+``` r
 # install.packages("devtools")
-devtools::install_github("2DegreesInvesting/r2dii.data")
+devtools::install_github("RMI-PACTA/r2dii.data")
 ```
 
-[How to raise an issue?](https://2degreesinvesting.github.io/posts/2020-06-26-instructions-to-raise-an-issue/)
+[How to raise an
+issue?](https://2degreesinvesting.github.io/posts/2020-06-26-instructions-to-raise-an-issue/)
```
Comment (jacobvjk) [inline README.md:44]: this is one of the blog posts we do not migrate? Is it not weird to have users get instructions on how to raise issues with 2DII?

File: README.Rmd [hunk 2]
```diff
@@ -44,7 +44,7 @@ Or install the development version of r2dii.data with something like this:
 
 ```r
 # install.packages("devtools")
-devtools::install_github("2DegreesInvesting/r2dii.data")
+devtools::install_github("RMI-PACTA/r2dii.data")
 ```
 
 [How to raise an issue?](https://2degreesinvesting.github.io/posts/2020-06-26-instructions-to-raise-an-issue/)
```
Comment (jacobvjk) [inline README.Rmd:50]: same as below

**Review by jacobvjk — COMMENTED** [review]
see brief questions before I approve

File: README.md (hunk 1 above)
Comment (jdhoffa) [reply README.md:44]: Unfortunately we elected not to migrate the 2dii blog to RMI, so in the spirit of consistency I just link it to the still existing blog post. 

I think this brings up a larger point as to if we want to start a similar blog at RMI

Comment (jacobvjk) [reply README.md:44]: Fair point. I guess the likelihood of people submitting issues to 2dii instead of RMI based on this is rather low

**Review by jacobvjk — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (jdhoffa) [reply README.md:44]: I think I will just remove this honestly... it is not that hard to raise an issue

**Review by jacobvjk — APPROVED** [review]
still lgtm

**Review by cjyetman — APPROVED** [review]

**Pull request merged**